ui/components/app/confirmation/footer/footer.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '../../../component-library';
+import {
+  BlockSize,
+  Display,
+  FlexDirection,
+} from '../../../../helpers/constants/design-system';
+
+const Footer = ({
+  cancelText = 'Cancel',
+  confirmText = 'Confirm',
+  onCancel,
+  onConfirm,
+}) => {
+  return (
+    <Box
+      display={Display.Flex}
+      flexDirection={FlexDirection.Row}
+      gap={4}
+      padding={4}
+      width={BlockSize.full}
+    >
+      <Button
+        variant={ButtonVariant.Secondary}
+        size={ButtonSize.Lg}
+        width={BlockSize.SixTwelfths}
+        onClick={onCancel}
+      >
+        {cancelText}
+      </Button>
```

Comment:
the hover style does not seem to match the current confirmation footer hover styles

https://github.com/MetaMask/metamask-extension/assets/20778143/1dc5a9f4-523f-4674-b315-2c35016544d3

Reply:
This is the default behaviour of the secondary button. @georgeweiler is there a way to disable this hover behaviour? Can't seem to find anything in the storybooks.

Reply:
This is the correct hover behaviour the design is most likely outdated